kitsune/questions/jinja2/questions/mobile/question_list.html

```diff
@@ -17,6 +17,13 @@
 {% set headline = title %}
 {% set return_url = url('questions.home') %}
 
+{% set canonical_url = canonicalize(viewname='questions.list', product_slug=product_slug)|urlparams(None, request.GET) %}
+{% if questions.number > 1 %}
+  {% set canonical_url = canonical_url|urlparams(page=questions.number) %}
+{% endif %}
+
+{% set meta = (('robots', 'noindex'),) %}
```

Comment:
we do want robots to index, right?

Reply:
The desktop version of this template has `{% set meta = (('robots', 'noindex'),) %}`, so I figured that should be kept for mobile.

(I don't know the history of this decision, but if desktop is not indexing and mobile hasn't been indexing, probably best to keep that going.)